plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -757,6 +788,143 @@ public ConnectorTableHandle beginDelete(ConnectorSession session, ConnectorTable
         throw new TrinoException(NOT_SUPPORTED, "This connector only supports delete where one or more partitions are deleted entirely");
     }
 
+    @Override
+    public void createView(ConnectorSession session, SchemaTableName viewName, ConnectorViewDefinition definition, boolean replace)
+    {
+        HiveIdentity identity = new HiveIdentity(session);
+        Map<String, String> properties = ImmutableMap.<String, String>builder()
+                .put(PRESTO_VIEW_FLAG, "true")
+                .put(PRESTO_VERSION_NAME, trinoVersion)
```

Comment:
what's the purpose of `PRESTO_VERSION_NAME` here and in hive connector? If it is just for debugging, should we add a marker mentioning iceberg connector to uniquely identify this codepath? 

I don't feel that strong either way. But this also helps easily separate iceberg/hive connector views in future if requirement comes up. on the other had, it also makes sense to see them as just "presto views".

Reply:
good idea. i've opened https://github.com/trinodb/trino/pull/8586 for this